tasks/ghost-os.yml

```diff
@@ -4,3 +4,9 @@
   template: src=etc/init/ghost.conf.j2 dest=/etc/init/ghost.conf
   notify: restart ghost
   tags: [ghost]
```

Comment:
I would like some global ansible config "am I upstart, or am I systemd" but was not able to locate one...   wonder how other people deal with this.